### PR TITLE
Fix golint issues in pkg/apis/core

### PR DESCRIPTION
**Pull request by hvaara**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
This PR fixes golint issues in `pkg/apis/core`

**Which issue(s) this PR fixes**:
Ref #68026

**Special notes for your reviewer**:
None

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

/sig testing
/priority backlog
/area code-organization

### CODE REVIEW

**Comment by hvaara** [comment]
@gmarek @ixdy
Friendly ping - can you PTAL?

File: pkg/apis/core/helper/qos/qos.go [hunk 1]
```diff
@@ -14,7 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// NOTE: DO NOT use those helper functions through client-go, the
+// Package qos provides helper functions related to QoS.
+// NOTE: DO NOT use these helper functions through client-go, the
 // package path will be changed in the future.
```
Comment (liggitt) [inline pkg/apis/core/helper/qos/qos.go]: can drop this NOTE, this package is not accessible from client-go

Comment (hvaara) [reply pkg/apis/core/helper/qos/qos.go]: Thanks for the feedback @liggitt! I'll fix this very soon

**Pull request closed without merging**